cds_ils/importer/providers/cds/rules/journal.py

```diff
@@ -108,20 +106,30 @@ def publisher(self, key, value):
     return clean_val("b", value, str, req=True)
 
 
-@model.over("_children", "(^362__)")
-def children_records(self, key, value):
-    """Translates fields related to children record types."""
-    _migration = self["_migration"]
-    _electronic_items = _migration.get("electronic_items", [])
-    if key == "362__":
-        _electronic_items.append({"subscription": clean_val("a", value, str)})
-    _migration.update(
-        {
-            "electronic_items": _electronic_items,
-        }
-    )
+@model.over("electronic_volumes_description", "(^362__)")
+def electronic_volumes_description(self, key, value):
+    """Translates electronic volumes description field."""
+    description = clean_val("a", value, str, req=True)
+    if description.endswith('-'):
+        description = description[:-1]
+    return description
```

Comment:
Does it make sense to also add a `strip`?
```suggestion
    return description.strip()
```
So that as in the test `v 1 (1992) -` becomes `v 1 (1992)`